resources/nats/values.yaml

```diff
@@ -316,7 +316,12 @@ appProtocol:
 k8sClusterDomain: cluster.local
 
 # Add labels to all the deployed resources
-commonLabels: {}
+commonLabels:
+  app.kubernetes.io/component: nats-manager
+  app.kubernetes.io/created-by: nats-manager
+  app.kubernetes.io/managed-by: nats-manager
+  app.kubernetes.io/part-of: nats-manager
+  control-plane: nats-manager
```

Comment:
I did apply the following labels:

```diff
-      app.kubernetes.io/instance: eventing
+      app.kubernetes.io/instance: nats-manager
-      app.kubernetes.io/name: nats
+      app.kubernetes.io/name: nats-manager
```

because they will case the label selectors to change which we don't want for the time being.